### PR TITLE
[ARKit] Update bindings to Xcode 12.3 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: tests/introspection/ApiProtocolTest.cs [hunk 1]
```diff
@@ -334,6 +334,8 @@ protected virtual bool Skip (Type type, string protocolName)
 				case "PKShareablePassMetadata":
 				// Xcode 12.2
 				case "VSAccountApplicationProvider": // Conformance not in headers
+				// Xcode 12.2
```
Comment (spouliot) [inline tests/introspection/ApiProtocolTest.cs:337]: 12.3 ?

File: tests/introspection/ApiSelectorTest.cs [hunk 2]
```diff
@@ -193,6 +193,13 @@ protected virtual bool Skip (Type type, string selectorName)
 				if (Runtime.Arch == Arch.SIMULATOR)
 					return true;
 				break;
+			case "ARBodyTrackingConfiguration":
+			case "ARGeoTrackingConfiguration":
```
Comment (spouliot) [inline tests/introspection/ApiSelectorTest.cs:197]: works on `ARWorldTrackingConfiguration` ?

**Review by spouliot — APPROVED** [review]

File: tests/introspection/ApiSelectorTest.cs (hunk 2 above)
Comment (dalexsoto) [reply tests/introspection/ApiSelectorTest.cs:197]: The test doesn't complain there, yeah that was a bit weird but device is fine so 🤷‍♂️

File: tests/introspection/ApiProtocolTest.cs (hunk 1 above)
Comment (dalexsoto) [reply tests/introspection/ApiProtocolTest.cs:337]: indeed

```suggestion
				// Xcode 12.3
```

**Review by tj-devel709 — APPROVED** [review]

**Pull request merged**